scripts/windows/common.ps1

```diff
@@ -0,0 +1,28 @@
+Set-PSDebug -Trace 1
+$MY_SCRIPT_DIR = Split-Path $script:MyInvocation.MyCommand.Path
+
+function extra_cflags_build_arg() {
```

Comment:
This is useless on windows because it is flags for gcc optimization for building collectd.

Reply:
leaving this in per our discussion